android-webrtc/URLVideoCapturer/README.md

```diff
@@ -0,0 +1,12 @@
+# URL Video Capturer for WebRTC
+
+This folder contains an Android library project that implements a WebRTC VideoCapturer for video playback from a URL. You can use it just like any other VideoCapturer in WebRTC. For example, you can use this implementation along with [Amazon Kinesis Video Streams Android WebRTC SDK](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-android) in order to add video streams from an RTSP camera into the WebRTC session.
```

Comment:
/nit Streaming from an IP camera is through the Real time Streaming Protocol (RTSP) URL provided by the camera (or media source).

Reply:
This solution shows how to enable streaming  IP cameras with any Android device to any WebRTC viewer  using AWS Kinesis Video Streams Android WebRTC.

Reply:
Done.